### PR TITLE
[stable/elasticsearch] Make pod annotations configurable for PodSecurityPolicy

### PR DESCRIPTION
**Pull request by spiarh**
Signed-off-by: Ludovic Cavajani <lcavajani@suse.com>

#### What this PR does / why we need it:
I think it is good to make the annotations configurable finally so an user can add his specifics. Moreover, it is safer if apparmor is not enabled for example.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/elasticsearch/values.yaml [hunk 1]
```diff
@@ -21,6 +21,15 @@ serviceAccounts:
 ##
 podSecurityPolicy:
   enabled: false
+  annotations: {}
```
Comment (rendhalver) [inline stable/elasticsearch/values.yaml:24]: Should we set some defaults for this?
Does it need any defaults or are the base settings ok?

Comment (spiarh) [reply stable/elasticsearch/values.yaml:24]: I think we should leave this configuration to the user, it's difficult to set some defaults as each k8s environment can be very different from one to an other.

Comment (rendhalver) [reply stable/elasticsearch/values.yaml:24]: That would leave these unconfigured.
Could we set this to the defaults from above?

Comment (spiarh) [reply stable/elasticsearch/values.yaml:24]: is it a problem to leave it unconfigured ? Even if most of them should work in almost all environment...

Comment (rendhalver) [reply stable/elasticsearch/values.yaml:24]: If they are different for each security provider we should put a note to that effect.

**Pull request merged**